src/js/sentio/chart/chart_donut.js

```diff
@@ -0,0 +1,566 @@
+sentio.chart.donut = sentio_chart_donut;
+
+function sentio_chart_donut() {
+	'use strict';
+
+	// Layout properties
+	var _id = 'donut_' + Date.now();
+	var _width = 460;
+	var _height = 300;
+	var _radius = 100;
+	var _innerRadiusOffset = 30;
```

Comment:
These are not used. I'd rather you use 'radius' and 'innerRadius' as the two parameters that dictate how the donut is drawn than use donutWidth. We may also want to rename this whole thing to 'circular_chart' or something since you can probably trivially change it between donut and pie mode.